examples/faith/pegasus_workflow_conf.ini

```diff
@@ -0,0 +1,137 @@
+[lalapps_inspinj]
```

Comment:
I think there is a question as to whether the flashy new faithsim code should use the creaky old injection script. But I think adding support for the pycbc injection generator could come later. It would also be good to be able to pregenerate injection files ... I wonder if we couldn't just use the workflow.injection module as it's used in the all-sky search? This then lets us give multiple injection files, in whatever way that module understands, which could be used to make multiple faithsims as needed??

Reply:
I think this all makes sense Ian, however I propose we do all that in a later PR.